ci-operator/config/oVirt/csi-driver/oVirt-csi-driver-ovirt-csi-driver-master.yaml

```diff
@@ -0,0 +1,41 @@
+base_images:
```

Comment:
this file should be named `oVirt-csi-driver-master.yaml `

Reply:
so that's a bug in `make jobs`
changing manually.

Reply:
scratch that, this is the config.

Reply:
its not bug. The tool though that your branch is named `ovirt-csi-driver-master` because of the bad naming.

Reply:
In addition, make jobs is not generating the ci-operator configs, but it reads from them.